css/elements/button.css

```diff
@@ -0,0 +1,220 @@
+    border-color: transparent;
+}
+
+.btn-primary:hover{
+    color: var(--white);
+    background-color: var(--btn-dark-primary);
+    border-color: transparent;
+}
+
+.btn-primary[disabled]:hover{
+    color: var(--white);
+    background-color: var(--btn-primary);
+    border-color: transparent;
+}
+
+.btn-secondary{
+    color: var(--white);
+    background-color: var(--btn-secondary);
+    border-color: transparent;
+}
+
+.btn-secondary:hover{
+    color: var(--white);
+    background-color: var(--btn-dark-secondary);
+    border-color: transparent;
+}
+
+.btn-secondary[disabled]:hover{
+    color: var(--white);
+    background-color: var(--btn-secondary);
+    border-color: transparent;
+}
+
+/* outlined buttons */
+.btn-outline-success{
+    color: var(--btn-success);
+    border-color: var(--btn-success);
+}
+
+.btn-outline-success:hover{
+    color: var(--white);
+    background-color: var(--btn-success);
+    border-color: var(--btn-success);
+}
+
+.btn-outline-success[disabled]:hover{
+    color: var(--btn-success);
+    background-color: transparent;
+    border-color: var(--btn-success);
+}
+
+.btn-outline-danger{
+    color: var(--btn-danger);
+    border-color: var(--btn-danger);
+}
+
+.btn-outline-danger:hover{
+    color: var(--white);
+    background-color: var(--btn-danger);
+    border-color: var(--btn-danger);
+}
+
+.btn-outline-danger[disabled]:hover{
+    color: var(--btn-danger);
+    background-color: transparent;
+    border-color: var(--btn-danger);
+}
+
+.btn-outline-primary{
+    color: var(--btn-primary);
+    border-color: var(--btn-primary);
+}
+
+.btn-outline-primary:hover{
+    color: var(--white);
+    background-color: var(--btn-primary);
+    border-color: var(--btn-primary);
+}
+
+.btn-outline-primary[disabled]:hover{
+    color: var(--btn-primary);
+    background-color: transparent;
+    border-color: var(--btn-primary);
+}
+
+.btn-outline-secondary{
+    color: var(--btn-secondary);
+    border-color: var(--btn-secondary);
+}
+
+.btn-outline-secondary:hover{
+    color: var(--white);
+    background-color: var(--btn-secondary);
+    border-color: var(--btn-secondary);
+}
+
+.btn-outline-secondary[disabled]:hover{
+    color: var(--btn-secondary);
+    background-color: transparent;
+    border-color: var(--btn-secondary);
+}
+
+/* icon button */
+.icon-btn{
+    display: inline-flex;
+    align-items: center;
+}
+
+.icon-btn span:last-child{
+    margin-left: var(--size-6);
+}
+
+/* button rounded */
+.btn-rounded {
+    border-radius: var(--radius-full);
+}
+
+/* button sizes */
+.btn-lg{
+    font-size: var(--size-11);
+}
+
+.btn-sm{
+    font-size: var(--size-8);
+}
+
+/* button floating */
+.btn-fab{
+    color: var(--white);
+    width: var(--size-16);
+    height: var(--size-16);
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    padding: var(--size-0);
+}
+
+.btn-fab-nbg{
+    color: initial;
+    border: var(--size-0);
+}
```

Comment:
The class Name **.btn-fab** could have been better, Could you explain what styles this class applies?.

Reply:
The classes **.btn-fab** and **.btn-fab-nbg**reperesents floating action button with background and without background color and I agree for class naming could be better.

I have changed class names to **.btn-floating** and **.btn-floating-wbg**.